test/linter.js

```diff
@@ -4,23 +4,71 @@
 'use strict';
 var
   execSync = require('child_process').execSync,
-  utils = require('./util/utils');
+  utils = require('./util/utils'),
+  fs = require('fs'),
+  path = require('path');
 
-describe('AutoRest Linter validation:', function () {
-  let configsToProcess = utils.getConfigFilesChangedInPR();
-  // Useful when debugging a test for a particular swagger. 
-  // Just update the regex. That will return an array of filtered items.
-  // configsToProcess = ['specification/sql/resource-manager/readme.md'];
-  for (const config of configsToProcess) {
+function getTagsFromConfig(config){
+  // get hold of all tags and their corresponding input files using the literate config tool
+  const result = execSync(`node node_modules/@microsoft.azure/literate/dist/app.js all --file=${path.resolve(config)}`);
```

Comment:
it'd be nice to write a comment with an example of what result looks like (the output of the tool)